setup.py

```diff
@@ -208,13 +208,12 @@ def get_git_sha() -> str:
         "starrocks": ["starrocks>=1.0.0"],
         "doris": ["pydoris>=1.0.0, <2.0.0"],
     },
-    python_requires="~=3.9",
+    python_requires="~=3.10",
```

Comment:
This should also be reverted given that:

> The `python_requires` field is used in the setup.py file of a Python package to specify the minimum version(s) of Python required to install and use the package.

and 3.9 would be the minimum version.